app/frontend/src/queryKeys.ts

```diff
@@ -40,3 +40,7 @@ export const referencesKey = (
   userId: number,
   type: "received" | "given" | "all"
 ) => ["references", { type, userId }];
+
+export type FriendRequestType = "Sent" | "Received";
+export const friendRequestKey = (type: FriendRequestType) =>
+  `friendRequests${type}`;
```

Comment:
```suggestion
  ["friendRequests", type];
```
Second param could even be an object, since it's "filters".